src/omero_zarr/raw_pixels.py

```diff
@@ -256,39 +253,28 @@ def plate_to_zarr(plate: omero.gateway._PlateWrapper, args: argparse.Namespace)
                 row_group = root.require_group(row)
                 col_group = row_group.require_group(col)
                 field_group = col_group.require_group(field_name)
-                n_levels, axes = add_image(img, field_group, cache_dir=cache_dir)
-                add_multiscales_metadata(field_group, axes, n_levels)
+                add_image(img, field_group, cache_dir=cache_dir)
                 add_omero_metadata(field_group, img)
                 # Update Well metadata after each image
                 col_group.attrs["well"] = {"images": fields, "version": VERSION}
                 max_fields = max(max_fields, field + 1)
             print_status(int(t0), int(time.time()), count, total)
 
         # Update plate_metadata after each Well
-        plate_metadata["wells"] = [{"path": x} for x in well_paths]
-        plate_metadata["field_count"] = max_fields
-        root.attrs["plate"] = plate_metadata
+        write_plate_metadata(
```

Comment:
What is the rationale for overwriting the plate metadata on each well?

Reply:
When exporting a large plate, I have found it useful to be able to view it in vizarr before it's complete.
Or if you cancel the export, or it fails at any point, you still have something you can view.